### ISSUE TITLE
Configuring `gradleDist.rootUrlMapper` doesn't work anymore using closure

### ISSUE THREAD

**Issue by tudortimi** [denis-zhdanov/custom-gradle-dist-gradle-plugin#13]
The following code doesn't work anymore:

```
gradleDist {
    rootUrlMapper = { version, type ->
        "https://artifactory.example.org/artifactory/gradle-distributions/gradle-${version}-${type}.zip"
    }
}
```

I think this breaks since https://github.com/harmonhsoft-tech/custom-gradle-dist-gradle-plugin/commit/ca751a2d6331d8b88f9ad3bd5ac74f07bab94763, because the type of `rootUrlMapper` was changed to `Property<...>` (https://github.com/harmonhsoft-tech/custom-gradle-dist-gradle-plugin/commit/ca751a2d6331d8b88f9ad3bd5ac74f07bab94763#diff-60d2b21177bb7d5377540492aeffff8bb7b16ed959c4304fa125c2130ad2b571R12).

**Comment by denis-zhdanov** [comment]
it's `customDistributionFileNameMapper` now. Documentation is updated for the same as well

**Comment by tudortimi** [comment]
`rootUrlMapper` is for defining where to download the base Gradle distribution. `customDistributionFileNameMapper` is for defining the file name of the resulting custom distribution.

Users still want to be able to configure the source URL. There's no example in the documentation on how to do this.

**Comment by denis-zhdanov** [comment]
ah, right, thanks

**Comment by denis-zhdanov** [comment]
fixed in `1.14.0`
